gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -715,3 +836,46 @@ def __init__(
 
   def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
+
+
+class WriteToVcf(PTransform):
+  """A PTransform for writing to VCF files."""
+
+  def __init__(self,
+               file_path,
+               num_shards=1,
+               compression_type=CompressionTypes.AUTO,
+               headers=None):
+    """Initialize a WriteToVcf PTransform.
+
+    Args:
+      file_path: The file path to write to. The files written will begin
+        with this prefix, followed by a shard identifier (see num_shards). The
+        file path should include the file extension (i.e. ".vcf", ".vcf.gz",
+        etc).
+      num_shards: The number of files (shards) used for output. If not set, the
+        service will decide on the optimal number of shards.
+        Constraining the number of shards is likely to reduce
+        the performance of a pipeline.  Setting this value is not recommended
+        unless you require a specific number of output files.
+      compression_type: Used to handle compressed output files. Typical value
+        for VCF files is CompressionTypes.UNCOMPRESSED. If set to
+        CompressionTypes.AUTO, file_path's extension will be used to detect
+        compression.
+      headers: A list of VCF meta-information lines describing the INFO and
+        FORMAT entries in each record and a header line describing the column
+        names. These lines will be written at the beginning of the file.
+    """
+    self._file_path = file_path
+    self._num_shards = num_shards
+    self._compression_type = compression_type
+    self._header = ''.join(headers) if headers else None
```

Comment:
Should this be '\n'.join?

Reply:
Hmm, how about calling strip() on each header and then joining with '\n'?